### PR TITLE
fix: Adding missing examples.

### PR DESCRIPTION
**Pull request by domderen**

### CODE REVIEW

**Comment by domderen** [comment]
I also made changes to the python runtime docker image & switched the python sdk to use threaded http server.

**Comment by domderen** [comment]
I'm not sure what is going on with that broken example test for `106-git-python`. I ran it locally on my machine, and it passes just fine. I'm not sure why it fails in Github Actions. Any ideas how to debug it?

File: Dockerfile [hunk 1]
```diff
@@ -88,8 +88,10 @@ RUN mkdir /.cache /.local
 ADD runtimes/python3-9 /workspace
 RUN chown -R 9653 /.cache /.local /workspace
 WORKDIR /workspace
+RUN apk add git
```
Comment (alexec) [inline Dockerfile:91]: I don't want to add git to this image. It increases the vuln surface and will create future work to keep patching.

Comment (alexec) [reply Dockerfile:91]: ok, so it looks like we really need this for pip

**Pull request merged**